plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -128,14 +128,18 @@ func (b *backend) pathVenafiIssue(ctx context.Context, req *logical.Request, dat
 		return nil, err
 	}
 	if role == nil {
-		return logical.ErrorResponse(fmt.Sprintf("unknown role: %s", roleName)), nil
+		return nil, fmt.Errorf("unknown role: %s", roleName)
 	}
 
 	if role.KeyType == "any" {
-		return logical.ErrorResponse("role key type \"any\" not allowed for issuing certificates, only signing"), nil
+		return nil, fmt.Errorf("role key type \"any\" not allowed for issuing certificates, only signing")
```

Comment:
```suggestion
		return nil, fmt.Errorf(`role key type "any" not allowed for issuing certificates, only signing`)
```

You can use backtick as a string delimiter and avoid having to escape the double quotes (ie: ` \"any\"`)

Reply:
Thank you for the suggestion. Already done by latest commit.